<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortWidget</class>
 <widget class="QWidget" name="SerialPortWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>639</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SerialPortWidget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Serial Port</string>
    </attribute>
    <widget class="QComboBox" name="flowControlBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Bits</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshPortsButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Refresh Ports</string>
     </property>
    </widget>
    <widget class="QPushButton" name="disconnectButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
    <widget class="QComboBox" name="stopBitsBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Baud Rate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Bits</string>
     </property>
    </widget>
    <widget class="QComboBox" name="serialPortsBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>41</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="dataBitsBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>141</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="parityBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>241</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRateBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Flow Control</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>45</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="driveInfoTab">
    <attribute name="title">
     <string>Drive Info</string>
    </attribute>
    <widget class="QTextBrowser" name="deviceInfoText">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>40</y>
       <width>311</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Drive Content</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Drive Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Free Space</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>256</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>256</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="actionsTab">
    <attribute name="title">
     <string>Actions</string>
    </attribute>
    <widget class="QTextEdit" name="fileNameTextField">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>451</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Content</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="readFileCheckbox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Read File</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="messageReceived">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>290</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="deleteFileCheckbox">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete File</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="writeFileCheckbox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Write File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendAction">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>File Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="contentTextField">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>154</y>
       <width>451</width>
       <height>121</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="updateFileCheckbox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Update File</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
